bot/exts/evergreen/bookmark.py

```diff
@@ -37,27 +104,39 @@ async def bookmark(
             await ctx.send(embed=embed)
             return
 
-        embed = discord.Embed(
-            title=title,
-            colour=Colours.soft_green,
-            description=target_message.content
-        )
-        embed.add_field(name="Wanna give it a visit?", value=f"[Visit original message]({target_message.jump_url})")
-        embed.set_author(name=target_message.author, icon_url=target_message.author.avatar_url)
-        embed.set_thumbnail(url=Icons.bookmark)
-
-        try:
-            await ctx.author.send(embed=embed)
-        except discord.Forbidden:
-            error_embed = discord.Embed(
-                title=random.choice(ERROR_REPLIES),
-                description=f"{ctx.author.mention}, please enable your DMs to receive the bookmark",
-                colour=Colours.soft_red
+        def event_check(reaction_: discord.Reaction, user_: discord.Member) -> bool:
+            """Make sure that this reaction is what we want to operate on."""
+            return (
+                # Conditions for a successful pagination:
+                all((
+                    # Reaction is on this message
+                    reaction_.message.id == reaction_message.id,
+                    # User has not already bookmarked this message
+                    user_.id not in bookmarked_users,
+                    # Reaction is the `BOOKMARK_EMOJI` emoji
+                    str(reaction_.emoji) == BOOKMARK_EMOJI,
+                    # Reaction was not made by the Bot
+                    user_.id != self.bot.user.id
+                ))
             )
-            await ctx.send(embed=error_embed)
-        else:
-            log.info(f"{ctx.author} bookmarked {target_message.jump_url} with title '{title}'")
-            await ctx.message.add_reaction(Emojis.envelope)
+        await self.action_bookmark(ctx.channel, ctx.author, target_message, title)
+
+        # Keep track of who has already bookmarked, so users can't spam reactions and cause loads of DMs
+        bookmarked_users = [ctx.author.id]
+        reaction_message = await self.send_reaction_embed(ctx.channel, target_message)
+
+        while True:
+            try:
+                _, user = await self.bot.wait_for("reaction_add", timeout=TIMEOUT, check=event_check)
+            except asyncio.TimeoutError:
+                log.debug("Timed out waiting for a reaction")
+                break  # No reactions for the last `TIMEOUT` seconds
```

Comment:
Just to be consistent with the rest of this file.
```suggestion
                log.debug("Timed out waiting for a reaction")
                # No reactions for the last `TIMEOUT` seconds
                break
```